media-libs/libggigcp/libggigcp-1.0.2-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Libggi extension for advanced color and palette handling"
+HOMEPAGE="https://ibiblio.org/ggicore/packages/libggigcp.html"
+SRC_URI="mirror://sourceforge/ggi/${P}.src.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=media-libs/libggi-2.2.2"
+DEPEND="${RDEPEND}"
+
+DOCS=( ChangeLog README doc/TODO doc/libggigcp-functions.txt \
+	doc/libggigcp-libraries.txt doc/libggigcp-structures.txt \
+	doc/libggigcp.txt doc/colors.faq doc/colors2.faq )
```

Comment:
you can use `doc/{libggigcp{,-libraries,-structures,functions}}` `doc/colors{,2}.faq` to shorten the list